datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -72,6 +72,16 @@ use super::get_output_ordering;
 /// Execution plan for scanning one or more Parquet partitions
 #[derive(Debug, Clone)]
 pub struct ParquetExec {
+    /// Override for `Self::with_pushdown_filters`. If None, uses
```

Comment:
I also added this back in so that the overrides set directly on a `ParquetExec` do not affect the global configuration options